src/neptune/new/internal/operation.py

```diff
@@ -185,7 +189,9 @@ def from_dict(data: dict) -> "AssignArtifact":
 class UploadFile(Operation):
 
     ext: str
-    file_path: str
+    file_path: str = None
+    tmp_file_name: str = None
+    # `clean_after_upload` is for backward compatibility and should be removed in the future
     clean_after_upload: bool = False
 
     @classmethod
```

Comment:
Maybe pass `OperationStorage` instead of `upload_path`?

Reply:
Ok, why not.